### PR TITLE
Smaller tab height and css changes

### PR DESCRIPTION
**Pull request by robinschmid**
We waste too much space in the windows  especially for tabs

- smaller height of tabs
- smaller check boxes
- outline for controls in dark mode as some blend to much with the bg

### CODE REVIEW

File: src/main/resources/themes/MZmine_dark.css [hunk 1]
```diff
@@ -60,9 +95,9 @@
 }
 
 /** Controls **/
-.button, .combo-box, .choice-box, .check-box > .box  {
+.button, .combo-box, .choice-box, .check-box > .box {
   -fx-background-color: -control-bg;
-  -fx-border-color: -control-bg;
+  -fx-border-color: -highlight-dark;
```
[the remark this answers is not in the document]
Comment (SteffenHeu) [reply src/main/resources/themes/MZmine_dark.css:100]: or the border color?

File: src/main/resources/themes/MZmine_light.css [hunk 2]
```diff
@@ -1,46 +1,79 @@
...
-
-  -selection-unfocused: derive(-fx-base, 10%);
-
-  -cell-odd-bg: -list-bg;
-  -cell-even-bg: derive(-list-bg, 2%);
-  -cell-selected-bg: derive(-control-bg, 20%);
-
-  -text-color: -fx-font-color;
+  -pane-bg-match: -fx-base
+;
```
Comment (SteffenHeu) [inline src/main/resources/themes/MZmine_light.css:36]: ?

**Review by SteffenHeu — COMMENTED** [review]
I don't like the smaller tabs at all. Makes everything look cramped.

**Comment by robinschmid** [comment]
I dont like that they take away so much space. that's unnecessary.
We can increase it a little bit but we should not have to many insets 
image

**Comment by robinschmid** [comment]
I have reverted the weird formatting. I had to exclude CSS files from the formatter for that. 
Also on your request I increased the tab height a little bit. 
image

**Comment by robinschmid** [comment]
Corinna and I like the dark blue borders. matches the highlighted color
image

**Comment by SteffenHeu** [comment]
did you push all the changes? The selected tab should be the tallest

**Pull request closed without merging**